broScripts/parse_pcap.py

```diff
@@ -54,6 +54,31 @@ def summary_stats():
 def is_person_empty(field):
     return field == '' or field == '-' or field == '<>' or field == '(empty)' or field == 'undisclosed'
 
+def parseLine(line):
+    lstOfTups = []
+
+    while True:
+        if "('" not in line or "')" not in line or "','" not in line:
+            return lstOfTups
+        startParen = line.index("('")
+        endParen = line.index("')")+1
+        comma = line.index("','") + 1
+
+        header = line[startParen+2:comma-1]
+        value = line[comma+2:endParen-1]
+
+        # remove duplicate escaping from Python
+        # matches the way eval deals with quotes
+        header.replace("\\\\\"", "\\\"")
+        value.replace("\\\\\"", "\\\"")
+
+        header.replace("\\\\\'", "\\\'")
+        value.replace("\\\\\'", "\\\'")
```

Comment:
The lines 72, 73, 75, and 76 are included so that the parser matches the output of eval(). I compared the output of eval and the output of the parser without those 4 lines and if those lines are not present, this parser has an extra backslash in front of the quotations.